src/Microsoft.DotNet.ProjectModel/Project.cs

```diff
@@ -84,6 +84,8 @@ public string ProjectDirectory
 
         public bool IsTestProject => !string.IsNullOrEmpty(TestRunner);
         
+        public bool OverrideIsRunnable { get; set; }
```

Comment:
Turns out this can't be fixed.  The "compilerOptions" object that is returned from GetCompilerOptions is a new object every time.  So if we set properties on it, they will be lost the next time GetCompilerOptions is invoked.

---
In reply to: [71799001](https://github.com/dotnet/cli/pull/3911#discussion_r71799001) [](ancestors = 71799001,71798696,71797149,71797075)